src/reader/mod.rs

```diff
@@ -77,6 +77,19 @@ macro_rules! configure_methods {
 
         /// Changes whether mismatched closing tag names should be detected.
         ///
+        /// Note, that start and end tags [should match literally][spec], they cannot
+        /// have different prefixes even if both prefixes resolves to the same namespace.
+        /// The
```

Comment:
This should be either "The XML" or "This XML" or "This:"

Reply:
Also "both prefixes resolves" -> "both prefixes resolve"

Reply:
Done ("The XML")!